flixel/tweens/FlxTween.hx

```diff
@@ -825,4 +829,30 @@ class FlxTweenManager extends FlxBasic
 		while (_tweens.length > 0)
 			remove(_tweens[0]);
 	}
+	/**
+	 * Immediately updates all tweens of type PERSIST or ONESHOT through their endings, triggering their onComplete callbacks.
+	 * 
+	 * Note: if they haven't yet begun, this will first trigger their onStart callback.
+	 * 
+	 * Note: their onComplete callbacks are triggered in the next frame.  To trigger them immediately, call `FlxTween.manager.update(0);` after this function.
```

Comment:
Would you have it so we call update(0) in the library, instead?  It's been a while so I forget the reason (if there was one) for adding a hint rather than just doing it in the library, which would have been easier...I'm trying to think if there's some case you'd want to get it to the end but have the callback occur on the next frame.  I know that for at least one usage I needed it to occur this frame, because after the onComplete possible set up new tweens and timers, I needed to have them skip to their endings, too, in a cascade.  Maybe I just didn't want to assume this was needed, but couldn't think of a use case then, either...?